lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,34 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      callback: callbacks,
+      onMessage: router.onMessage,
+      transport: transports.WEBSOCKET,
+      requestId: requestId,
+      lastDataBatchTime: router.getLatestBatchTimeStamp,
+    };
+    var jobObject = {};
+    jobObject.channel = requestId;
+    activeSocket.send(JSON.stringify(jobObject));
+  }
+
+  function unRegisterLiveTail(requestId){
+    activeSocket.close();
+    authToken = null;
+    clearInterval(keepAliveInterval);
+    delete knownComputations[requestId];
+  }
+
   return {
     authenticate: authenticate,
     execute: execute,
     stop: stop,
-    disconnect: disconnect
+    disconnect: disconnect,
+    registerLiveTail: registerLiveTail,
+    unRegisterLiveTail: unRegisterLiveTail
```

Comment:
Since the protocol calls it `stoplivetail` I would make the function name match that.